engine/storage/integration-test/test/org/apache/cloudstack/storage/test/FakePrimaryDataStoreDriver.java

```diff
@@ -90,7 +96,7 @@ public void makeTakeSnapshotSucceed(boolean success) {
     }
 
     @Override
-    public void revertSnapshot(SnapshotInfo snapshot, AsyncCompletionCallback<CommandResult> callback) {
+    public void revertSnapshot(SnapshotInfo snapshotOnImageStore, SnapshotInfo snapshotOnPrimaryStore, AsyncCompletionCallback<CommandResult> callback) {
```

Comment:
This appears to be the default implementation from the IDE. I realize that it predates this patch, but should method do more? If not, can we strike the IDE generated comment to avoid future confusion?